compiler/codegen/OMRInstruction.hpp

```diff
@@ -136,6 +136,12 @@ class OMR_EXTENSIBLE Instruction
     */
    virtual uint8_t *generateBinaryEncoding() = 0;
 
+   /**
+    * Expand this instruction into multiple instructions prior to binary encoding.
+    * Returns the last instruction in the expansion.
```

Comment:
Apologies for being pedantic, but could you add proper Doxygen tags for this comment please (e.g., `@brief`, `@returns`, etc.)

Reply:
No problem. Done in a1398596c7e866a7e44a02dcaed632d9276a35e7.